pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["setuptools>=41", "setuptools-git-versioning<2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scvi-distributed"
+authors = [
+  { name="Yerdos Ordabayev", email="yordabay@broadinstitute.org" },
+  { name="Mehrtash Babadi", email="mehrtash@broadinstitute.org" },
+]
+description = "Machine learning library for single-cell data analysis"
+readme = "README.rst"
+requires-python = ">=3.10"
+license = {file = "LICENSE.md"}
+classifiers = [
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: BSD License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: 3",
+]
+
+dependencies = [
+  "anndata",
```

Comment:
I recently had a problem with the CAS repo because I didn't specify versions for the packages. Just another suggestion is to provide specific versions or version ranges.